src/biblebot/setup_utils.py

```
@@ -234,9 +251,25 @@
+        if line.strip().startswith("ExecStart="):
+            new_lines.append(f"ExecStart={exec_cmd} --config {default_config}")
+        else:
+            new_lines.append(line)
+    service_content = "\n".join(new_lines) + "\n"
+
     # Write service file
     try:
-        get_user_service_path().write_text(service_template)
+        get_user_service_path().write_text(service_content)
         print(f"Service file created at {get_user_service_path()}")
         return True
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)